_posts/en/2019-10-01-btse-exchange-operation.md

```diff
@@ -0,0 +1,30 @@
+---
+title: 'Field Report: Effective, secure exchange operation using Bitcoin technologies at BTSE'
+permalink: /en/btse-exchange-operation/
+name: 2019-10-01-btse-exchange-operation
```

Comment:
Will want to make sure we update this date if we do not push live today.

Reply:
I suggest you add a `FIXME` comment to update the date. That way Travis will fail its checks so we won't accidentally merge without updating the date.